pkg/client.go

```diff
@@ -21,28 +21,28 @@ type SecretClient struct {
 	Manager SecretStoreManager
 }
 
-func (sc SecretClient) GetSecret(key string) (string, error) {
-	value, err := sc.Manager.GetValue(key)
+func (sc SecretClient) GetSecrets(keys ...string) (map[string]string, error) {
```

Comment:
I like these changes. Can we just add some GoDocs stating how the errors are returned to the caller? For example, do we throw an error if one of the keys are not found. This also goes for the other functions for the `ServiceClient` as they accept more than one key as parameters.

Just a thought...We can also map an error to the key and leave it up to the client to perform the check, or a combination of both where we return an error and map errors to the invalid keys.